testsuite/features/profiles/github_runner/Docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM localhost:5002/opensuse/leap:15.4
```

Comment:
Just out of curiousity, why do we not use Leap 15.6 or at least 15.5?

Reply:
no idea. I am using what we are already using at the other build profiles.